src/lib/backend-api.ts

```diff
@@ -46,7 +46,7 @@ export interface IStageEntryController {
 
     getReviews(): Promise<Review[]>;
     addReview(spec: ReviewSpec): Promise<Review>;
-    review(userId: number): IReviewController;
+    review(reviewerId: number): IReviewController;
```

Comment:
I think the reason was that it can be used like this:
```ts
BackendController.getInstance().project(2).stage(34).paper(2).review(4).get();
```
With more descriptive names, this would look like the following code
```ts
BackendController.getInstance()
        .getProjectController(2)
        .getStageController(34)
        .getPaperController(2)
        .getReviewController(4)
        .get();
```

In my opinion, either way is fine. Do you want me to change this?

Reply:
Ok, i understand why the current solution looks nice. Probably we should stay with it for more readable code. Your demonstration showed that this is probably more useful than self-describing method names (if you agree, resolve it).